app/assets/javascripts/modules/vulnerabilities/components/summary.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="vulnerabilities-summary">
+    <p class="highlight">Portus security scanner has detected <strong>{{ vulnerabilities.length }}</strong> vulnerabilities</p>
```

Comment:
I'd change "Portus" by "The". Although since clair is a security scanner and we support multiple ones, maybe it would be better to simply say "We have detected ..." (because Portus doesn't have a security scanner, but it supports multiple ones theoretically).